api/api.go

```diff
@@ -247,6 +247,17 @@ type CloudBackupCreateRequest struct {
 	Full bool
 }
 
+type CloudBackupGroupCreateRequest struct {
```

Comment:
Move all new structs to api.proto

Reply:
Create a new ticket to track this effort separately #516